configure/CONFIG_SITE.Common.linux-x86_64

```diff
@@ -0,0 +1,2 @@
+# Set this to YES if the hidapi package is installed for USB devices on Linux.
+HAVE_HIDAPI=YES
```

Comment:
There is need for this file, the HIDAPI library is no longer needed on Linux.  Please remove.

Reply:
file removed